apps/dashboard/src/app/form.tsx

```diff
@@ -35,6 +37,29 @@ type InputFieldContext<T extends FieldValues> = {
 }
 type InputProducerResult<F extends FieldValues> = FC<InputFieldContext<F>>
 
+export function createMultipleSelectInput<F extends FieldValues>({
+  ...props
+}: Omit<MultiSelectProps, "error">): InputProducerResult<F> {
+  return function FormSelectInput({ name, state, control }) {
+    return (
+      <Controller
+        control={control}
+        name={name}
+        render={({ field }) => (
+          <MultiSelect
+            {...props}
+            error={state.errors[name] && <ErrorMessage errors={state.errors} name={name} />}
+            onChange={field.onChange}
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
```

Comment:
Always use ts-expect-error instead of tsignore
```suggestion
            // @ts-expect-error
```